### ISSUE TITLE
Cannot open centrafuge

### ISSUE THREAD

**Issue by tyler489** [EnigmaticaModpacks/Enigmatica6#4686]
### Modpack Version

1.1.2 Expert

### Describe your issue.

Cannot open centrafuge, or the multiblock centrafuge when processing pcb combs.

### Have you modified the modpack?

No

### Did the issue happen in singleplayer or on a server?

Server

**Comment by NillerMedDild** [comment]
Did you have something in your offhand? :)

**Comment by tyler489** [comment]
I do not believe so, is there something specific that doesnt work with offhands?

**Comment by MuteTiefling** [comment]
A lot of things don't work properly if you have an item in your offhand. Guis not opening, things not activating, etc

**Comment by NillerMedDild** [comment]
Let us know if you manage to reproduce the issue with an empty offhand :)
